test/release-ttk.ps1

```diff
@@ -19,7 +25,7 @@ Copy-Item "..\..\arm-ttk\arm-ttk" -Destination ".\template-tests" -Recurse
 $releaseFiles += ".\template-tests"
 $releaseFiles = $releaseFiles -ne "..\..\arm-ttk/arm-ttk"
 Compress-Archive -DestinationPath "AzTemplateToolkit.zip" -Path $releaseFiles -Force
-Remove-Item ".\template-tests" -Recurse
+Remove-Item ".\template-tests" -Recurse -Force
```

Comment:
This is because the Mac adds .DS_Store hidden files to things on disk, and PowerShell refuses to delete folders with hidden files.